### PR TITLE
[TAP-5174] Webspoon integration into Pentaho base code - pentaho-kett…

### PR DESCRIPTION
**Pull request by angel-ramoscardona**
…le/ui

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 18m 31s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl ui
```
---
:ok_hand: All tests passed!
 
Tests run: **1203**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/pdi/job/PR-8118/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by e-cuellar — APPROVED** [review]

File: ui/src/main/java/org/pentaho/di/ui/spoon/trans/LogBrowser.java [hunk 1]
```diff
@@ -130,7 +130,7 @@ public void run() {
                 lastNr = Math.min( lastNr, lastLogId.get() + MAX_NR_LOG_LINES_CHUNK );
 
                 List<KettleLoggingEvent> logLines =
-                  KettleLogStore.getLogBufferFromTo( childIds, true, lastLogId.get(), lastNr );
+                  KettleLogStore.getLogBufferFromTo( childIds, false, lastLogId.get(), lastNr );
```
Comment (rmansoor) [inline ui/src/main/java/org/pentaho/di/ui/spoon/trans/LogBrowser.java:133]: @angel-ramoscardona  this change to disable all general log lines in kettle log store. Do you know what is the reason for this change ?

File: ui/src/main/java/org/pentaho/di/ui/core/widget/ConditionEditor.java [hunk 2]
```diff
@@ -733,7 +741,7 @@ public void widgetSelected( SelectionEvent e ) {
         break;
       case AREA_OPERATOR:
         Menu mPop = new Menu( widget );
-        MenuItem miDown = new MenuItem( mPop, SWT.CASCADE );
+        MenuItem miDown = new MenuItem( mPop, SWT.PUSH );
```
Comment (rmansoor) [inline ui/src/main/java/org/pentaho/di/ui/core/widget/ConditionEditor.java:744]: @angel-ramoscardona 
What is the impact on the PDI. Have you seen any difference
